scripts/import/import_EPMC.pl

```diff
@@ -589,7 +670,144 @@ sub update_meta{
+
+    my $wrong_title_sth = $dba->dbc->prepare(qq[ select publication_id, title from publication where title like '%Errata%' or title like '%Erratum%' or title like '%In This Issue%' or title like '%Oral abstracts%' or title like '%Oral Presentations%' or title like '%Proffered paper%' or title like '%Subject Index%' or title like '%Summaries of Key Journal Articles%' or title like '%This Month in The Journal%' or title like 'Index%' or title like '%Table of Contents%' or title like '%Not Available%' ]);
+
+    $not_available_sth->execute()||die;
+    my $not_available_titles = $not_available_sth->fetchall_arrayref();
+
+    $title_sth->execute()||die;
+    my $title_brackets =  $title_sth->fetchall_arrayref();
+
+    $wrong_title_sth->execute()||die;
+    my $wrong_title = $wrong_title_sth->fetchall_arrayref();
+
+    # Delete publications with title 'Not Available'
+    if(defined $not_available_titles->[0]->[0]){
+      print $report "\nPublications with 'not available' title deleted (publication_id, title, variation_id):\n";
+      remove_publications($dba, $report, $not_available_titles);
+    } 
+
+    # Clean brackets from publication title 
+    my $count_titles = 0;
+    if (defined $title_brackets->[0]->[0]){
+      print $report "Publications with brackets in the title - clean the brackets:\n";
+      foreach my $l (@{$title_brackets}){
+        $count_titles += 1;
+        my $pub_id = $l->[0];
+        my $title = $l->[1];
+        print $report "$pub_id\t$title\n";
+        $title =~ s/^\[//;
+        $title =~ s/\]//;
+
+        my $pub_clean_sth = $dba->dbc()->prepare(qq[ update publication set title = ? where publication_id = $pub_id ]);
+        $pub_clean_sth->execute($title);
+      }
+    }
+
+    # Delete publications with not acceptable titles
+    if(defined $wrong_title->[0]->[0]){
+      print $report "\nPublications with not acceptable title deleted (publication_id, title, variation_id):\n";
+      remove_publications($dba, $report, $wrong_title);
+    }
+
+    # If there is publications without title, delete them 
+    if($n_title_null != 0){
+      my $title_null_sth = $dba->dbc->prepare(qq[ select publication_id from publication where title is null or title = '' ]); 
+      my $title_null_sth->execute();
+      my $titles_null = $title_null_sth->fetchall_arrayref();
+ 
+      if(defined $titles->[0]->[0]){
+        print $report "\nPublications without title deleted (publication_id, title, variation_id):\n";
+        remove_publications($dba, $report, $titles_null);
+      }
+    }
+
+    $publication_count = count_rows($dba, 'publication');
+    $citation_count    = count_rows($dba, 'variation_citation');
+
+    print $report "\nTotal publications after cleaning:\t$publication_count\n";
+    print $report "Total citations after cleaning:\t$citation_count\n";
+}
+
+# Delete publication from publication and variation_citation tables
```

Comment:
If the removed publication was the only publication for a variant, the citation evidence may need to be removed from variation_feature.evidence_attribs.

Reply:
Thanks! Checked.